grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -67,7 +68,8 @@ class InventoryItemController {
 
             // now populate the rest of the commmand object
             inventoryService.getStockCardCommand(cmd, params)
-            [commandInstance: cmd]
+            def totalQuantityAvailableToPromise = productAvailabilityService.getQuantityAvailableToPromise(cmd.product, cmd.warehouse)
```

Comment:
Let's move this inside getStockCardCommand() method if possible just so it happens any time we get the stock card command object.